backend/utils/chunked_tts.py

```
@@ -50,6 +51,44 @@
         "u.s",
         "u.s.a",
         "u.k",
+        # Russian
+        "т.д",   # и т.д. (и так далее)
+        "т.п",   # и т.п. (и тому подобное)
+        "т.е",   # т.е. (то есть)
```

Ruff (0.15.11)

[warning] String contains ambiguous `е` (CYRILLIC SMALL LETTER IE). Did you mean `e` (LATIN SMALL LETTER E)?

(RUF001)